pkgvars

```diff
@@ -1,4 +1,4 @@
 pkgname=citus
 pkgdesc='Citus (Open-Source)'
 pkglatest=10.0.6.citus-1
-versioning=fancy
```

Comment:
Why did you remove the empty line in this file?

Reply:
Added again